<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Users\Daniel\Documents\projetos\DbC\src\Dbc.UnitTests\bin\Debug\DbC.UnitTests.dll" total="7" errors="0" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2011-07-23" time="21:07:51">
  <environment nunit-version="2.5.5.10112" clr-version="4.0.30319.1" os-version="Microsoft Windows NT 6.0.6001 Service Pack 1" platform="Win32NT" cwd="c:\Program Files\Microsoft Visual Studio 10.0\Common7\IDE" machine-name="DANIEL-PC" user="Daniel" user-domain="Daniel-PC" />
  <culture-info current-culture="pt-BR" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Users\Daniel\Documents\projetos\DbC\src\Dbc.UnitTests\bin\Debug\DbC.UnitTests.dll" executed="True" result="Success" success="True" time="0.002" asserts="0">
    <results>
      <test-suite type="Namespace" name="DbC" executed="True" result="Success" success="True" time="0.002" asserts="0">
        <results>
          <test-suite type="Namespace" name="UnitTests" executed="True" result="Success" success="True" time="0.002" asserts="0">
            <results>
              <test-suite type="TestFixture" name="AndAssertionTest" executed="True" result="Success" success="True" time="0.001" asserts="0">
                <results>
                  <test-case name="DbC.UnitTests.AndAssertionTest.WhenComposedOfATrueAssertionAndAFalseAssertion_ThenValidated_ThrowsExceptionWithFalseAssertionMessage" executed="True" result="Success" success="True" time="0.001" asserts="2" />
                  <test-case name="DbC.UnitTests.AndAssertionTest.WhenComposedOfTwoFalseAssertions_ThenValidated_ThrowsExceptionWithBothMessages" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                  <test-case name="DbC.UnitTests.AndAssertionTest.WhenComposedOfTwoTrueAssertions_ThenValidated_DoNotThrowsException" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                </results>
              </test-suite>
              <test-suite type="TestFixture" name="AssertionTest" executed="True" result="Success" success="True" time="0.000" asserts="0">
                <results>
                  <test-case name="DbC.UnitTests.AssertionTest.WhenAnAssertionIsInvalid_AndDoesNotHaveMessage_ThrowsExceptionWithDefaultMessage" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                  <test-case name="DbC.UnitTests.AssertionTest.WhenAnAssertionIsInvalid_AndHasMessage_ThrowsExceptionWithSpecifiedMessage" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                  <test-case name="DbC.UnitTests.AssertionTest.WhenAnAssertionIsValid_DoNotThrowsException" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="DbC.UnitTests.AssertionTest.WhenTwoAssertionsAreUsedInALogicalAnd_ReturnsAnObjectOfAssertionType" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>